### PR TITLE
Add "developer mode" and test last cap and dbus docs

### PR DESCRIPTION
**Pull request by keszybz**
This add opt-in automated tests for two things which we occasionally forget to update:
CAP_LAST_CAP and the generated dbus doc stubs.

### CODE REVIEW

File: man/meson.build [hunk 1]
```diff
@@ -198,16 +198,14 @@ run_target(
 
 ############################################################
 
-if dbus_docs.length() > 0
```
Comment (yuwata) [inline man/meson.build]: If `xsltproc` is not found, then `dbus_docs` is empty.

**Review by yuwata — CHANGES_REQUESTED** [review]
CI fails with the following.
```
ERROR: Command cannot have '@INPUT@', since no input files were specified
```

File: man/meson.build (hunk 1 above)
Comment (keszybz) [reply man/meson.build]: Ooops. Apparently there was a reason for this conditional. I removed that commit now.

**Review by yuwata — APPROVED** [review]
LGTM.

**Comment by keszybz** [comment]
test-path failed, but in an unrelated way:
```
--- stderr ---
-.slice: Failed to migrate controller cgroups from /user.slice/user-1000.slice/session-1.scope, ignoring: No such file or directory
path-exists.service: Failed to create cgroup /user.slice/user-1000.slice/session-1.scope/path-exists.service: Permission denied
path-exists.service: Succeeded.
path-exists.service: Failed to create cgroup /user.slice/user-1000.slice/session-1.scope/path-exists.service: Permission denied
path-exists.service: Succeeded.
path-exists.path: Succeeded.
-.slice: Failed to migrate controller cgroups from /user.slice/user-1000.slice/session-1.scope, ignoring: No such file or directory
path-existsglob.service: Failed to create cgroup /user.slice/user-1000.slice/session-1.scope/path-existsglob.service: Permission denied
path-existsglob.service: Succeeded.
path-existsglob.service: Failed to create cgroup /user.slice/user-1000.slice/session-1.scope/path-existsglob.service: Permission denied
path-existsglob.service: Succeeded.
path-existsglob.path: Succeeded.
-.slice: Failed to migrate controller cgroups from /user.slice/user-1000.slice/session-1.scope, ignoring: No such file or directory
path-changed.service: Failed to create cgroup /user.slice/user-1000.slice/session-1.scope/path-changed.service: Permission denied
path-changed.service: Succeeded.
path-changed.service: Failed to create cgroup /user.slice/user-1000.slice/session-1.scope/path-changed.service: Permission denied
path-changed.service: Succeeded.
path-changed.path: Succeeded.
-.slice: Failed to migrate controller cgroups from /user.slice/user-1000.slice/session-1.scope, ignoring: No such file or directory
path-modified.service: Failed to create cgroup /user.slice/user-1000.slice/session-1.scope/path-modified.service: Permission denied
path-modified.service: Succeeded.
path-modified.service: Failed to create cgroup /user.slice/user-1000.slice/session-1.scope/path-modified.service: Permission denied
path-modified.service: Succeeded.
path-modified.path: Succeeded.
-.slice: Failed to migrate controller cgroups from /user.slice/user-1000.slice/session-1.scope, ignoring: No such file or directory
path-mycustomunit.service: Failed to create cgroup /user.slice/user-1000.slice/session-1.scope/path-mycustomunit.service: Permission denied
path-mycustomunit.service: Failed to attach to cgroup /user.slice/user-1000.slice/session-1.scope/path-mycustomunit.service: No such file or directory
path-mycustomunit.service: Failed at step CGROUP spawning /bin/true: No such file or directory
path-mycustomunit.service: Main process exited, code=exited, status=219/CGROUP
path-mycustomunit.service: Failed with result 'exit-code'.
Test timeout when testing path-unit.path
-------
```

**Pull request merged**